src/main/java/io/github/moulberry/notenoughupdates/listener/ItemTooltipRngListener.java

```diff
@@ -324,6 +339,24 @@ private void runsRequired(
 		}
 	}
 
+	private int parseAmountFromTooltip(List<String> tooltip) {
+		Map<Integer, Integer> tierToAmount = new HashMap<>();
+
+		for (String line : tooltip) {
+			Matcher amountMatcher = TIER_AMOUNT.matcher(line);
+			if (amountMatcher.matches()) {
+				try {
+					// group(2) will get the lower bound of number of drops that is dropped from each of the tiers (i.e. if "2 to 4" we get 2)
```

Comment:
I think this is a good spot to use 2 variables with helpful names instead of having a complicated looking one-liner and a comment on top.